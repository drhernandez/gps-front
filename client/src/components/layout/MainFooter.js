import React from "react";
import PropTypes from "prop-types";
import {Container, Row} from "shards-react";

const MainFooter = ({ contained, menuItems, copyright }) => (
  <footer className="main-footer d-flex p-2 px-3 bg-white border-top">
    <Container fluid={contained}>
      <Row className="d-flex align-items-center h-100">
        <span className="copyright ml-auto my-auto mr-2">{copyright}</span>
      </Row>
    </Container>
  </footer>
);

MainFooter.propTypes = {
  /**
   * Whether the content is contained, or not.
   */
  contained: PropTypes.bool,
  /**
   * The menu items array.
   */
  menuItems: PropTypes.array,
  /**
   * The copyright info.
   */
  copyright: PropTypes.string
};

MainFooter.defaultProps = {
  contained: false,
  copyright: "Copyright Â© 2019 DesignRevision"
};

export default MainFooter;
