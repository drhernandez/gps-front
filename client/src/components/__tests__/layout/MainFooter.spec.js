import React from 'react'
import renderer from 'react-test-renderer'

import MainFooter from '../../layout/MainFooter'

it('snapshot test', () => {
  const component = renderer.create(
    <MainFooter />
  )

  expect(component.toJSON()).toMatchSnapshot()
})