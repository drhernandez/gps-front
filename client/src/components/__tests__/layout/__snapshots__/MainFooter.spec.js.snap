// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<footer
  className="main-footer d-flex p-2 px-3 bg-white border-top"
>
  <div
    className="container"
  >
    <div
      className="d-flex align-items-center h-100 row"
    >
      <span
        className="copyright ml-auto my-auto mr-2"
      >
        Copyright Â© 2019 DesignRevision
      </span>
    </div>
  </div>
</footer>
`;
